pkg/test/istio.io/tasks/security/secure-gateway-sds/secure_gateway_sds_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package securegatewaysds
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/istio.io/examples"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+var (
+	inst istio.Instance
+	g    galley.Instance
+	p    pilot.Instance
+)
+
+func TestMain(m *testing.M) {
+	// Integration test for the ingress SDS Gateway flow.
+	framework.
+		NewSuite("sds_ingress_gateway_test", m).
```

Comment:
We want the test names to match the doc names.
The URL is `auth-sds`, so use the same name here. And rename this test's directory into `auth-sds`, and this file into `auth-sds_test.go`.

Reply:
All the directory name, file name and test name are updated. Thanks.